bekeplar/iReporter
# Ch application structure and cognitive complexity

## What does this PR do?

- Restructuring the project structure
- Reduce cognitive complexity
##  Description of Task to be completed?

- Separating user concerns from incidents concern
- Putting a similar line of code to a given location for usability for example code in views which is related
##  How should this be manually tested?
On this Pull request navigate to the files changed tab to see the various files changed
## Screenshots
![changes](https://user-images.githubusercontent.com/42211777/51340054-d7ab4400-1a42-11e9-8fa7-808e965b8850.PNG)
![changes1](https://user-images.githubusercontent.com/42211777/51340064-da0d9e00-1a42-11e9-8525-82f76fc831b8.PNG)
![changes2](https://user-images.githubusercontent.com/42211777/51340074-e0037f00-1a42-11e9-96cb-748f5570b0d7.PNG)
![changes3](https://user-images.githubusercontent.com/42211777/51340079-e2fe6f80-1a42-11e9-9be6-e3f801acd1e0.PNG)
## Story attached
[#163286622]
,
@wasswa-derick @solnsubuga @3Nakajugo, @masete, @nearjay06, @JamesMudidi, @silver,
@sanya-kenneth @neelxie @frankopkusianwar @AtamaZack @ManuelDominic, @kalsmic
